docs/gui/menus.md

```diff
@@ -273,12 +273,19 @@ A `MenuProvider` is an interface that contains two methods: `#createMenu`, which
 A `MenuProvider` can easily be created using `SimpleMenuProvider`, which takes in a method reference to create the server menu and the title of the menu.
 
 ```java
-// In some implementation
-NetworkHooks.openScreen(serverPlayer, new SimpleMenuProvider(
+/*
+  In some implementation
+  args:
+   IForgeServerPlayer pPlayer , BlockPos pPos
+  use:
+  {@link IForgeServerPlayer#openMenu(MenuProvider, BlockPos)}
+  {@link IForgeServerPlayer#openMenu(MenuProvider, Consumer)}
+*/
+pPlayer.openMenu(new SimpleMenuProvider(
   (containerId, playerInventory, player) -> new MyMenu(containerId, playerInventory),
-  Component.translatable("menu.title.examplemod.mymenu")
-));
+  Component.translatable("menu.title.examplemod.mymenu"),pPos);
```

Comment:
```suggestion
  Component.translatable("menu.title.examplemod.mymenu"), menuPos);
```

No need to use parchment's mapping with extra 'p'. It doesn't serve a purpose here. It would probably be good to explain that the position is where the menu is opened and checked from.